src/TrackerCouncil.Smz3.Tracking/TrackingServices/TrackerModeService.cs

```diff
@@ -137,4 +139,42 @@ public void StopShaktoolMode(float? confidence = null)
             ToggledShaktoolMode?.Invoke(this, new TrackerEventArgs(confidence));
         });
     }
+
+    public void EnableCheats(float? confidence = null)
+    {
+        if (!CheatsEnabled)
+        {
+            CheatsEnabled = true;
+            Tracker.Say(x => x.Cheats.EnabledCheats);
+            CheatsToggled?.Invoke(this, new TrackerEventArgs(confidence));
+            AddUndo(() =>
+            {
+                CheatsEnabled = false;
+                ToggledShaktoolMode?.Invoke(this, new TrackerEventArgs(confidence));
```

Comment:
Uhm... I don't think this is the right event?

Reply:
lol I guess that's _probably_ not correct.